app/Draft.js

```diff
@@ -52,7 +52,7 @@ export default class Draft {
 		if (!id) {
 			throw new Error('An id is required to retrieve a draft from register.');
 		}
-
+        CurrentDraft=id;
```

Comment:
If I understood, the idea of having a `CurrentDraft` makes it that so only one draft can happen at a time. Is that correct?